runtime/parachains/src/builder.rs

```diff
@@ -440,6 +443,8 @@ impl<T: paras_inherent::Config> BenchBuilder<T> {
 					})
 					.collect();
 
+				let _ = validity_votes.drain(*num_votes as usize..);
```

Comment:
We can avoid making the signatures in the first place by just doing `group_validators.iter().take(num_votes).map(..` above

Reply:
Great point!